electron-react/remote.md

```diff
@@ -0,0 +1,39 @@
+# Connecting the UI to a Remote Daemon
```

Comment:
Lets check this in for this release but I think once it's out we should migrate it to the wiki instead.

Reply:
Makes sense